src/components/volunteers/Volunteer/index.tsx

```diff
@@ -58,30 +57,48 @@ export default function Volunteer({
           <Typography variant="h3" pt={2} pb={2}>
             {volunteer.firstName} {volunteer.lastName}
           </Typography>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
               Email:
             </Typography>
-            <Typography display="inline">{volunteer.email}</Typography>
+            <Typography display="inline" variant="h6">
+              {volunteer.email}
+            </Typography>
           </Box>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
```

Comment:
Please add `fontWeight: 'normal'` to all the vals in these pairs

(i.e. When displaying "Email: andlrutt@vols.utk.edu", `andlrutt@vols.utk.edu` should not be bold)

Reply:
This applies to RoleVerifications as well